### PR TITLE
Add vpc for sonar

### PR DESCRIPTION
**Pull request by Eli-Meitner**
Add support to attach vpc to codebuild (for access to  sonarqube on main account private)

### CODE REVIEW

File: modules/codepipeline/templates/package.yaml [hunk 1]
```diff
@@ -0,0 +1,220 @@
+AWSTemplateFormatVersion: 2010-09-09
```
Comment (ShakedYosef) [inline modules/codepipeline/templates/package.yaml]: Is this file needed? if no - please delete.

**Review by ShakedYosef — APPROVED** [review]

**Review by evgenygigi — APPROVED** [review]
